formats/service_manual_guide/frontend/examples/basic_with_related_discussions.json

```diff
@@ -17,6 +17,28 @@
       { "title" : "What is it, why it works and how to do it", "href" : "#what-is-it-why-it-works-and-how-to-do-it" }
     ]
   },
+  "links": {
+    "topics": [{
+      "content_id": "f08d0fbb-c14e-4c08-8260-a24c9eacceb4",
+      "title": "Agile delivery",
+      "base_path": "/service-manual/agile-delivery",
```

Comment:
I've checked with Tom and he says that you've agreed with publishing topics under `/service-manual` initially (at the bottom of [Tom's notes](https://docs.google.com/document/d/1-O_Zt_k7cjXm-ZTzMA12xDPQu2G5qzFCT8YIPaYzmAk/edit)). I think, among other reasons, we can't currently publish under `/topics` because the IA team has not allocated any topic to Service manuals. 

Tom also mentioned that you spoke about creating redirects when we migrate to collections publisher. 

Come over and talk to Tom if my answer is not very clear, I'm sure he can explain it better :)